eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -95,3 +95,4 @@ jobs:
           TestOptions: ${{ parameters.TestOptions }}
           TestParallelization: ${{ parameters.TestParallelization }}
           BuildParallelization: ${{ parameters.BuildParallelization }}
+          ContinueLatestBuildOnError: ${{ eq(variables['LatestJdkVersion'], variables['JavaTestVersion']) }}
```

Comment:
I don't think this syntax is correct as those variables aren't available at template compile type so they will always be empty.